samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AssessmentSettingsBean.java

```diff
@@ -1933,7 +1933,10 @@ public void setTransitoryExtendedTime(ExtendedTime newExTime) {
     public void addExtendedTime() {
         ExtendedTime entry = this.extendedTime;
         FacesContext context = FacesContext.getCurrentInstance();
-        if (new ExtendedTimeValidator().validateEntry(entry, context, this)) {
+        //check for duplicate users/groups
+        List<ExtendedTime> extendedTimesTempList = new ArrayList<ExtendedTime>(extendedTimes);
+        extendedTimesTempList.add(entry);
+        if (new ExtendedTimeValidator().validateEntries(extendedTimesTempList, context, this)) {
```

Comment:
this doesn't appear to change anything as the original call does the same thing, only the list is immutable?

Reply:
The orignal code calls ExtendedTimeValidator().validateEntry(entry, context, this), which validates only a single entry.